### PR TITLE
Posts get ordered when they're retrieved

### PR DESCRIPTION
**Pull request by derekdowling**

### CODE REVIEW

File: app/scripts/components/content.jsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ var Content = React.createClass({
         }
 
         if (this.props.isPost) {
-          comments.push(<ContentCreator post={this.props.data} forComment={!isPost} />);
+          comments.push(<ContentCreator key="comment-creator" post={this.props.data} forComment={!isPost} />);
```
Comment (derekdowling) [inline app/scripts/components/content.jsx:27]: This doesn't really need a unique key.

File: app/scripts/stores/post.js [hunk 2]
```diff
@@ -69,26 +69,39 @@ var PostStore = Reflux.createStore({
...
-            b = this.posts[b];
+    // sorts posts into reverse chronological order. Aka, newest first.
+    orderPosts: function (posts) {
+
+        var postArr = [];
+        if (posts.size == 0) {
+            return postArr;
+        }
+
+        for (var value of posts.values()) {
```
Comment (derekdowling) [inline app/scripts/stores/post.js:90]: Gotta love the "of" syntax.

**Comment by derekdowling** [comment]
@KarimBaaba this is good to go.

**Comment by xkb** [comment]
LGTM :+1:

**Pull request merged**